examples/dws.cpp

```diff
@@ -47,48 +46,48 @@ vector<string> split(const string& str, const string& delim)
 	{
 		start = end;
 		while (start < str.size() && (delim.find(str[start]) != string::npos))
-			start++;  // skip initial whitespace
+			start++; // skip initial whitespace
 		end = start;
 		while (end < str.size() && (delim.find(str[end]) == string::npos))
-			end++; // skip to end of word
-		if (end-start != 0)  // just ignore zero-length strings.
-			parts.push_back(string(str, start, end-start));
+			end++;			  // skip to end of word
```

Comment:
Changing spaces to tabs on comments after the code is weird.

Reply:
We should probably manually remove the extra spaces